quickstart/speech-translation/cpp-windows/README.md

```diff
@@ -0,0 +1,40 @@
+# Quickstart: Translate speech in C++ for Windows
+
+This sample demonstrates how to translate speech with C++ using the Speech SDK for Windows.
+See the [accompanying article](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-translate-speech-cpp-windows) on the SDK documentation page which describes how to build this sample from scratch in Visual Studio 2017.
+
+## Prerequisites
+
+* A subscription key for the Speech service. See [Try the speech service for free](https://docs.microsoft.com/azure/cognitive-services/speech-service/get-started).
+* A Windows PC with a working microphone.
+* [Microsoft Visual Studio 2017](https://www.visualstudio.com/), Community Edition or higher.
+* The **Desktop development with C++** workload in Visual Studio and the **NuGet package manager** component in Visual Studio.
+  You can enable both in **Tools** \> **Get Tools and Features**, under the **Workloads** and **Individual components** tabs, respectively.
+
+## Build the sample
+
+* **By building this sample you will download the Microsoft Cognitive Services Speech SDK. By downloading you acknowledge its license, see [Speech SDK license agreement](https://aka.ms/csspeech/license201809).**
+* [Download the sample code to your development PC.](../../README.md#get-the-samples)
```

Comment:
broken link, please fix in new all READMEs

Reply:
Hey Mark, to clarify, is the suggestion that **Recognition** samples stay directly under `/quickstarts` and that all subsequent features (**Translation** and eventually **TTS**) would follow this pattern `/quickstarts/feature-x`? I think that's fine for now, but may be confusing later.

Reply:
yes, that is the suggestion. Please keep the location of existing quickstarts, and, yes, I will look into cleaning up the structure later.

---
In reply to: [249870069](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/142#discussion_r249870069) [](ancestors = 249870069)